code/drasil-example/sglpend/stack.yaml

```diff
@@ -12,8 +13,10 @@ packages:
 - ../../drasil-lang
 - ../../drasil-metadata
 - ../../drasil-printers
+- ../../drasil-sysinfo
 - ../../drasil-theory
 - ../../drasil-utils
+- ../dblpend
```

Comment:
why does sglpend now depend on dblpend?

Reply:
It always did. `stack build` from `sglpend` directory was giving an error because it failed to find the local package, so I've added it. This change is from one of the two earlier PRs, however.

https://github.com/JacquesCarette/Drasil/blob/13ce53a5803fbd9ac3935c3ae9b03b3c2507cc11/code/drasil-example/sglpend/package.yaml#L16-L31